frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install.tsx

```diff
@@ -102,7 +104,15 @@ export const CreateAttachedDevicesCluster: React.FC<CreateAttachedDevicesCluster
   );
 };
 
+type NavUtils = {
+  getStep: (offset?: number) => number;
+  getParamString: (step: number, mode: number) => string;
+  getIndex: (searchSpace: any, search: string, offset?: number) => number;
+  getAnchor: (step: number, mode: number) => string;
+};
```

Comment:
Put it in commonplace and export from there and use it. Rather than defining multiple times.